packages/utilities/src/properties.ts

```diff
@@ -271,7 +271,9 @@ export function getNativeProps<T>(props: {}, allowedPropNames: string[], exclude
     (propName: string) => {
       return (
         (!excludedPropNames || excludedPropNames.indexOf(propName) < 0) &&
-        (propName.indexOf('data-') === 0 || propName.indexOf('aria-') === 0 || allowedPropNames.indexOf(propName) >= 0)
+        (propName.indexOf('data-') === 0 || propName.indexOf('aria-') === 0 || allowedPropNames.indexOf(propName) >= 0) &&
+        // tslint:disable-next-line:no-any
```

Comment:
I tried to fix the typing of `getNativeProps` to avoid this any, (primarily by casting props as `keyof T`) but it exposed further issues both up and down the calling hierarchy. `keyof` is technically of type `PropertyKey` which caused issues with the `filteredAssign`. It also exposed issues with calls into this function.